### PR TITLE
Changed references to serviceinnovationlab... to the new domain name.

### LINKED ISSUE #240: add family.services.govt.nz to allowed domains
in the reactjs app, probably in index.html, the app needs to be happy with this new domain in all the same places that it currently uses serviceinnovationlab.github.io

### PR DESCRIPTION
**Pull request by JacOng17**
Changed references to serviceinnovationlab.github.io to the new domain name (family.services.govt.nz).

Not too sure if that's all we need to do/or that I'm right.

closes #240

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
+    <meta http-equiv="Content-Security-Policy" content="connect-src 'self' ws://localhost:3000 https://catalogue.data.govt.nz https://api.addressfinder.io; style-src blob: https://api.addressfinder.io https://unpkg.com https://family.services.govt.nz 'unsafe-inline'; script-src 'self' https://api.addressfinder.io https://family.services.govt.nz https://unpkg.com 'unsafe-eval' 'unsafe-inline' 'unsafe-inline'; default-src 'none'; img-src 'self' data: https://a.tile.openstreetmap.org https://b.tile.openstreetmap.org https://c.tile.openstreetmap.org https://family.services.govt.nz https://unpkg.com; child-src 'none'; base-uri https://family.services.govt.nz; form-action https://family.services.govt.nz; object-src 'none'; manifest-src 'self';">
```
Comment (Br3nda) [inline public/index.html]: Could we make the app allow *both* for now? . Then we can keep testing while waiting for tbe new domain.

Comment (mermop) [reply public/index.html]: Should this be something determined by environment variables? There could be anything on localhost:3000 - that's probably not something that should be allowed in production!

Comment (Br3nda) [reply public/index.html]: Seems reasonable. But i don't know how.
Should we generate it at build time?

Comment (Br3nda) [reply public/index.html]: @lamorrison @JacOng17 Any ideas how to improve this and not have all the domains hard coded here?

**Pull request merged**